src/navigations/HomeBottomTabs.js

```diff
@@ -119,13 +119,15 @@ function HomeBottomTabs() {
   };
 
   return (
-    <SafeAreaView style={{flex: 1, backgroundColor: COLORS.white}}>
+    <SafeAreaView style={{flex: 1, backgroundColor: COLORS.almostBlack}}>
       <Loading visible={isLoadingFetchingChannelDetail} />
       <Tab.Navigator
         initialRouteName={getInitialRouteName()}
         tabBarOptions={{
-          inactiveTintColor: COLORS.blackgrey,
-          safeAreaInsets: {top: 0, bottom: 0, left: 0, right: 0}
+          inactiveTintColor: COLORS.grey410,
+          safeAreaInsets: {top: 0, bottom: 0, left: 0, right: 0},
+          activeBackgroundColor: COLORS.almostBlack,
+          inactiveBackgroundColor: COLORS.almostBlack
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `SafeAreaView` and `Tab.Navigator` components seem to be primarily focused on color adjustments. The background color of the `SafeAreaView` has been changed from white to almost black, and similar changes have been made in the `tabBarOptions` for both active and inactive backgrounds. The inactive tint color has also been updated. These changes appear to be part of a larger theme or style update across the application. As long as these color changes align with the overall design goals and maintain good contrast for accessibility, they look fine.
```diff
-    <SafeAreaView style={{flex: 1, backgroundColor: COLORS.white}}>
+    <SafeAreaView style={{flex: 1, backgroundColor: COLORS.almostBlack}}>
```
```diff
-          inactiveTintColor: COLORS.blackgrey,
+          inactiveTintColor: COLORS.grey410,
+          activeBackgroundColor: COLORS.almostBlack,
+          inactiveBackgroundColor: COLORS.almostBlack
```
Please ensure that these color changes are consistent throughout the app to maintain a cohesive user experience.


<!-- This is an auto-generated comment by OSS CodeRabbit -->